src/core/Cargo.toml

```diff
@@ -49,11 +50,14 @@ once_cell = "1.18.0"
 ouroboros = "0.18.3"
 piz = "0.5.0"
 primal-check = "0.3.1"
+rayon = { version = "1.8.1", optional = true }
 rkyv = { version = "0.7.44", optional = true }
 roaring = "0.10.3"
-rayon = { version = "1.8.1", optional = true }
+roots = "0.0.8"
 serde = { version = "1.0.196", features = ["derive"] }
 serde_json = "1.0.113"
+statrs = "0.16.0"
+streaming-stats = "0.2.3"
```

Comment:
I didn't look more deeply into them, but I guess we can't consolidate into one?

Reply:
`statrs` does implement a `statistics::Median` trait and `statistics::VarianceN` trait, but I don't know how do use them.

Reply:
upon digging a little further, `statrs` provides distributions (e.g. `ContinuousCDF`, `Normal`) that we need for ANI confidence intervals, while `streaming-stats` is used for `median` and `stddev` directly from a vector of abundances. I think we need both (or a different stats library entirely).